doc/releases/changelog-0.11.0.md

```diff
@@ -23,106 +26,104 @@
 
+* Three new compilation passes have been added to help convert Clifford + T gates to Pauli product measurements 
+  (PPMs) as prescribed in [arXiv:1808.02892](https://arxiv.org/abs/1808.02892v3). 
   [(#1499)](https://github.com/PennyLaneAI/catalyst/pull/1499)
   [(#1551)](https://github.com/PennyLaneAI/catalyst/pull/1551)
-  [(#1564)](https://github.com/PennyLaneAI/catalyst/pull/1564)
-
-  Supported gate conversions:
-    - H gate → PPR with (Z · X · Z)π/4
-    - S gate → PPR with (Z)π/4
-    - T gate → PPR with (Z)π/8
-    - CNOT → PPR with (Z ⊗ X)π/4 · (Z ⊗ 1)−π/4 · (1 ⊗ X)−π/4
-
-    Example:
-    ```python
-        @qjit(keep_intermediate=True)
-        @to_ppr
-        @qml.qnode(dev)
-        def circuit():
-            qml.H(0)
-            qml.S(1)
-            qml.T(0)
-            qml.CNOT([0, 1])
-            m1 = catalyst.measure(wires=0)
-            m2 = catalyst.measure(wires=1)
-            return m1, m2
-        circuit()
-    ```
-
-    The PPRs and PPMs are currently only represented symbolically. However, these operations are not yet executable on any backend since they exist purely as intermediate representations for analysis and potential future execution when a suitable backend is available.
-
-    Example MLIR Representation:
-    ```mlir
-      . . .
-        %0 = quantum.alloc( 2) : !quantum.reg
-        %1 = quantum.extract %0[ 1] : !quantum.reg -> !quantum.bit
-        %2 = qec.ppr ["Z"](4) %1 : !quantum.bit
-        %3 = quantum.extract %0[ 0] : !quantum.reg -> !quantum.bit
-        %4 = qec.ppr ["Z"](4) %3 : !quantum.bit
-        %5 = qec.ppr ["X"](4) %4 : !quantum.bit
-        %6 = qec.ppr ["Z"](4) %5 : !quantum.bit
-        %7 = qec.ppr ["Z"](8) %6 : !quantum.bit
-        %8:2 = qec.ppr ["Z", "X"](4) %7, %2 : !quantum.bit, !quantum.bit
-        %9 = qec.ppr ["Z"](-4) %8#0 : !quantum.bit
-        %10 = qec.ppr ["X"](-4) %8#1 : !quantum.bit
-        %mres, %out_qubits = qec.ppm ["Z"] %9 : !quantum.bit
-        %mres_0, %out_qubits_1 = qec.ppm ["Z"] %10 : !quantum.bit
-      . . .
-    ```
-
-* Commuting Clifford Pauli Product Rotation (PPR) operations to the end of a circuit, past non-Clifford PPRs, is now available through the :func:`~.catalyst.passes.commute_ppr` pass transform.
   [(#1563)](https://github.com/PennyLaneAI/catalyst/pull/1563)
+  [(#1564)](https://github.com/PennyLaneAI/catalyst/pull/1564)
+  [(#1577)](https://github.com/PennyLaneAI/catalyst/pull/1577)
   
-  A PPR is a rotation gate of the form :math:`\exp{iP \theta}`, where :math:`P` is a Pauli word (a product of Pauli operators). Clifford PPRs refer to PPRs with :math:`\theta = \tfrac{\pi}{4}`, while non-Clifford PPRs have :math:`\theta = \tfrac{\pi}{8}`.
+  These new compilation passes are currently only represented symbolically and are not yet executable 
+  on any backend. They exist purely as intermediate representations for analysis and potential future 
+  execution when a suitable backend is available.
```

Comment:
Compilation passes don't execute on backends, it's the PPM circuits that aren't executable:
```suggestion
  Note that programs in the PPM representation cannot yet be executed on available backends.
  The passes currently exist for analysis, but may PPMs program may become executable in the future
  when a suitable backend is available.
```